### PR TITLE
11.0 mig purchase commercial partner

### PR DESCRIPTION
**Pull request by andreasdsp**

### CODE REVIEW

File: purchase_commercial_partner/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Purchase Commercial Partner',
+    'version': '11.0.1.0.0',
+    'category': 'Purchase Management',
+    'license': 'AGPL-3',
+    'summary': "Add stored related field 'Commercial Supplier' on POs",
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```
Comment (nikul-serpentcs) [inline purchase_commercial_partner/__manifest__.py:12]: @andreasdsp Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

**Review by nikul-serpentcs — COMMENTED** [review]
Small Changes

File: purchase_commercial_partner/models/purchase.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    commercial_partner_id = fields.Many2one(
```
Comment (rousseldenis) [inline purchase_commercial_partner/models/purchase.py:10]: @andreasdsp Set readonly=True.

File: purchase_commercial_partner/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```
Comment (anandkansagra) [inline purchase_commercial_partner/__manifest__.py:1]: Please remove `# -*- coding: utf-8 -*-`

**Review by anandkansagra — CHANGES_REQUESTED** [review]

**Pull request closed without merging**